spec/support_specs/clockwork_spec.rb

```diff
@@ -4,7 +4,7 @@
 require './app/workers/clockwork_check'
 require './app/workers/send_applications_to_provider_worker'
 require './app/workers/decline_offers_by_default_worker'
-require './app/workers/send_chase_email_to_referees_worker'
+require './app/workers/send_chase_email_to_candidates_worker'
 require './app/workers/send_chase_email_to_providers_worker'
```

Comment:
Do you need to `require` the new workers here too? I guess not, in which case I wonder why we have these here at all.